Promises.xcodeproj/TestHost_Info.plist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
```suggestion
	<string>com.google.$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Reply:
This will not work unfortunately since the bundle_id is being set for the `ios_application` target in the BUILD file.